tests/benchmark/config.json

```diff
@@ -59,6 +59,12 @@
       "rows": 385817,
       "conn_id": "bigquery",
       "file_type": "ndjson"
+    },
+    {
+        "name": "few_kb",
+        "size": "45K",
+        "path": "../data/covid_overview.csv",
+        "rows": 732
```

Comment:
Does this not need `conn_id` key like other records.

nit: formatting/tabs seems off compared to other records

Reply:
@kaxil - the conn_id is optional for local files.